js/views/site.es6.js

```diff
@@ -16,6 +18,16 @@ function Site (ops) {
       [this.$whitelisttoggle, 'click', this._whitelistClick]
     ]);
 
+    // FIXME should be moved to the right place --- model?
```

Comment:
I think we should go with this for now so that 'beta' is showing the top blocked list and domain.

then fix it, and the # trackers, the whitelist status in a separate PR(s)